languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -19997,4 +19997,268 @@ DB Schenker
 Schenker AG
 Arda Güler
 Prime Reading
-Amazon Freevee
\ No newline at end of file
+Amazon Freevee
+Anna Márton
+Carol Levy
+Elizabeth Hussey
+Jane Grey
+Jane Leeves
+Jane Moraes
+Joan Gregson
+Joan Marsh
+Joan Martí i Alanis
+Joan Riviere
+Julia Görges
+Maria Koller
+Maria Melita
+Paula Grace
+Paula Borges
+Rose Parish
+Beni Johnson
+Jennifer Walton
+Marguerite Durand
+María Moliner
+Renee Walker
+Alexis Rose
+Josie Maran
+Leonora Fani
+Marnie Simpson
+María Gabriela
+Adrian Fernández
```

Comment:
This seems a misspelling in Spanish. Adrián Fernández: https://en.wikipedia.org/wiki/Adri%C3%A1n_Fern%C3%A1ndez

Reply:
Yes. That's true. Don't worry about them. We can fix them afterward.